hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -178,6 +178,19 @@ <h3 style="margin-top:40px"> NetCDF Header Info</h3>
             {% endfor %}
     {% endif %}
 
+
+    {# Add netcdf opendap button  #}
+    {% if opendap_link %}
+        {% for f in cm.files.all %}
+            {% if f.resource_file.name|slice:"-3:" == '.nc'%}
```

Comment:
@gantian127 do we need to do additional check here to see whether .nc file exists since the check is already done in your page processor which means if opendap_link is not empty, then .nc file already exists?